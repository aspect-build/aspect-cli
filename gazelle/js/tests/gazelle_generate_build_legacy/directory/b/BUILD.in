# gazelle:generation_mode update
