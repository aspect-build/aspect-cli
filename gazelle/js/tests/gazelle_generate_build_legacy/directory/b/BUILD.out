load("@aspect_rules_ts//ts:defs.bzl", "ts_project")

# gazelle:generation_mode update

ts_project(
    name = "b",
    srcs = [
        "b1/lib-b1.ts",
        "lib-b.ts",
    ],
    deps = ["//directory/a"],
)
